subgraphs/curve/deployments.ts

```diff
@@ -0,0 +1,54 @@
+export interface Contract {
+    address: string;
+    startBlock: number;
+}
+
+export interface NamedContract extends Contract {
+    name: string;
+}
+
+export interface StandalonePool extends NamedContract {
+    lpToken: string;
+}
+
+export interface ChainConfig {
+    grafting?: {
+        base: string;
+        block: number;
+    };
+    nativeToken: {
+        address: string;
+        symbol: string;
+        name: string;
+        decimals: number;
+    };
+    contracts: {
+        MainRegistry: Contract;
+        GaugeController: Contract;
+    };
+}
+
+export const DEPLOYMENTS: Record<string, ChainConfig> = {
+    mainnet: {
+        // grafting: {
```

Comment:
What's this?

Reply:
I can use this to deploy a subgraph telling the Graph Node to not start indexing it rfom scratch, but instead take a specific block and only start indexing from that one, keeping all the previously indexed entities